src/VIPServer.java

```diff
@@ -107,24 +107,6 @@ public void main(String[] args) throws HostFailureException, HostNotFoundExcepti
 					// SHOULD BE REPLACED BY
 					// job.carryOn();
 				} else {
-					if (!timer) {
```

Comment:
Ok, about point 1), if we implement like this, it will work. Using getClock to compute the delay between GATE job ends and Merge job starts. But in this case, we imply that the VIPServer knows how long the queuing time of Merge job is. I feel it is a little less realist. 

Maybe it is the name of process "Timer" makes confusion. In fact, the reason why i created a new process in GATE_DISCONNECT is to test whether Merge job really starts, if not, VIPServer waits for a retry delay and test again because if we dont do this test, a "BEGIN" may be sended to Merge job before merge process really starts, which may cause errors (a msg to a process not existing).
In this implementation, the VIPServer does not know the exact queuing time of Merge job. And in order to not block the main process VIPServer, i create a new process.